app/browser/tabs.js

```diff
@@ -134,6 +134,15 @@ const aboutTabUpdateListener = (tabId) => {
   }
 }
 
+const isOnAboutLocation = (url, search) => {
+  if (url == null) {
+    return false
+  }
+
+  const path = url.split('?')
+  return Array.isArray(path) && path[0] === search
```

Comment:
Nice! this is a great way to solve it 😄  For those who may not be aware, the link to the backup page is generated here:
https://github.com/brave/browser-laptop/blob/2d3fd8f19a09fb67a15115f2c073fa5ea8ffa976/app/browser/api/ledger.js#L248-L251

(Notice how the visible flag comes after the ?, which is what caused the original issue)